### ISSUE TITLE
Is there a way to have *codespell* reports on PR ?

### ISSUE THREAD

**Issue by acionca** [TheAxonLab/hcph-sops#255]
I love that *codespell* is catching typos when pushing/merging PRs but is there a way to have these reports before merging ? I believe that would be usefull to avoid making a new PR for a few typos.

**Comment by oesteban** [comment]
yes - updating this line https://github.com/TheAxonLab/hcph-sops/blob/ab490965640ef7d0cf64277994d8acb096a3991d/.github/workflows/tests.yml#L3

**Comment by acionca** [comment]
Cool, thanks – was there a reason to not have codespell run on PR ? And what should be put instead of `push` ?

**Comment by oesteban** [comment]
> Cool, thanks – was there a reason to not have codespell run on PR ? And what should be put instead of `push` ?

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

TL;DR `[push, pull-request]`
